### PR TITLE
[Issue 10579] Fix inputs to return a list of topic

### LINKED ISSUE #10579: Sinks API endpoint `getSinkInfo` does not properly return a list of topic inputs
**Describe the bug**
Currently, the [documented response shape](https://pulsar.apache.org/sink-rest-api/?version=2.7.1#operation/getSinkInfo) for `getSinkInfo` lists a node called `inputs` that should show a list of names of topics used as inputs for a given sink. Currently, requesting this endpoint returns a response where `inputs` always returns `null` instead of a list of topics.

**To Reproduce**
1. Create a tenant, namespace, and at least one topic
2. Create a new sink, setting the new topic(s) as topic input(s) to the sink
3. Request sink details via the `getSinkInfo` endpoint
4. See that `inputs` returns `null` instead of a list of topic(s) used to set up the sink.

**Expected behavior**
To receive a list of strings that match the topic names used during sink setup in step 2 of reproduction steps.

**Screenshots**

Sample request
```http
GET https://<baseUrl>/admin/v3/sink/tenant/default/elasticSearchSink
Authorization: <redacted>
Accept: application/json, text/plain, */*
```

Sample response
```http
HTTP/1.1 200 OK
Content-Type: application/json
Vary: Origin
Date: Thu, 13 May 2021 16:25:19 GMT
Content-Length: 1221
Connection: close

{
  "tenant": "tenant",
  "namespace": "default",
  "name": "elasticSearchSink",
  "className": "org.apache.pulsar.io.elasticsearch.ElasticSearchSink",
  "sourceSubscriptionName": null,
  "sourceSubscriptionPosition": null,
  "inputs": null,
  "topicToSerdeClassName": null,
  "topicsPattern": null,
  "topicToSchemaType": null,
  "topicToSchemaProperties": null,
  "inputSpecs": {
    "persistent://tenant/default/new-topic": {
      "schemaType": null,
      "serdeClassName": null,
      "schemaProperties": {},
      "receiverQueueSize": null,
      "regexPattern": false
    }
  },
  "maxMessageRetries": null,
  "deadLetterTopic": null,
  "configs": {
    "password": "<redacted>",
    "elasticSearchUrl": "<redacted>",
    "indexName": "<redacted>",
    "username": "<redacted>"
  },
  "secrets": null,
  "parallelism": 1,
  "processingGuarantees": "ATLEAST_ONCE",
  "retainOrdering": false,
  "resources": null,
  "autoAck": true,
  "timeoutMs": 5000,
  "negativeAckRedeliveryDelayMs": null,
  "archive": "builtin://elastic_search",
  "cleanupSubscription": null,
  "runtimeFlags": null,
  "customRuntimeOptions": "<redacted>"
}
```

**Desktop (please complete the following information):**
 - OS: Mac OSX Catalina 10.15.7

### PR DESCRIPTION
**Pull request by murong00**
### Motivation

Fixes #10579

### Modifications

Add the list of topic in field `SinkConfig.inputs`.

### CODE REVIEW

File: pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/SinkConfigUtils.java [hunk 1]
```diff
@@ -250,6 +250,7 @@ public static SinkConfig convertFromDetails(FunctionDetails functionDetails) {
         sinkConfig.setParallelism(functionDetails.getParallelism());
         sinkConfig.setProcessingGuarantees(FunctionCommon.convertProcessingGuarantee(functionDetails.getProcessingGuarantees()));
         Map<String, ConsumerConfig> consumerConfigMap = new HashMap<>();
+        List<String> inputs = new LinkedList<>();
```
Comment (eolivelli) [inline pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/SinkConfigUtils.java:253]: can we use ArrayList ?

**Review by eolivelli — APPROVED** [review]
LGTM

I left one minor comment, can you please take a look ?

File: pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/SinkConfigUtils.java (hunk 1 above)
Comment (murong00) [reply pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/SinkConfigUtils.java:253]: Sure, I will refine this later, thanks.

**Review by sijie — APPROVED** [review]

File: pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/SinkConfigUtils.java (hunk 1 above)
Comment (murong00) [reply pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/SinkConfigUtils.java]: Done.

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**